js/src/components/contact-information/store-address-card.js

```diff
@@ -122,12 +123,13 @@ export default function StoreAddressCard() {
 }
 
 /**
- * "Edit MC store address" Tracking event
+ * Trigger when store address edit button is clicked.
+ * Before `1.5.0` this name was used for tracking clicking "Edit in settings" to edit the WC address. As of `>1.5.0`, that event is now tracked as `edit_wc_store_address`.
  *
  * @event gla_edit_mc_store_address
  * @type {Object} TrackingEvent
```

Comment:
```suggestion
 * @type {Object}
```

Reply:
Avoiding 

`WARNING: The @type tag does not permit a description; the description will be ignored. File: store-address-card.js, line: 125`